apps/server/src/shared/domain/domainobject/ltitool.do.ts

```diff
@@ -1,8 +1,8 @@
-import { EntityId } from '@shared/domain/types';
 import { LtiPrivacyPermission, LtiRoleType } from '@shared/domain/entity/ltitool.entity';
+import { EntityId } from '@shared/domain/types';
 import { BaseDO } from './base.do';
 
-export class CustomLtiProperty {
+export class CustomLtiPropertyDO {
```

Comment:
Why you renamed this?

Reply:
because CustomLtiProperty is a property of the entity that had a collision somewhere.